package.json

```diff
@@ -1,8 +1,16 @@
 {
   "dependencies": {
+    "@rails/ujs": "^6.0.2-2",
     "@rails/webpacker": "5.0.1",
+    "bootstrap": "^4.4.1",
     "core-js": "^3.6.4",
-    "regenerator-runtime": "^0.13.5"
+    "expose-loader": "^0.7.5",
```

Comment:
This should be part of the `devDependencies`. We should bundle this into the production build.